packages/ui/src/views/repo/repo-summary/components/summary-panel.tsx

```diff
@@ -62,13 +68,24 @@ const SummaryPanel: FC<SummaryPanelProps> = ({
             <span className="text-13 text-foreground-4">Created {timestamp}</span>
           </>
         )}
+        <Spacer size={3} />
+        <Badge
+          size="md"
+          disableHover
+          borderRadius="full"
+          theme={!is_public ? 'muted' : 'success'}
+          className="w-[25%] items-center justify-center"
```

Comment:
why do we need a width on the badge? should it not be dynamic based on the content?